src/Administration/Resources/app/administration/src/module/sw-product/component/sw-product-variant-modal/sw-product-variant-modal.html.twig

```diff
@@ -37,7 +37,10 @@
                     @click="toggleFilterMenu"
                 >
                     {{ $tc('sw-product.variations.generatedFilterList') }}
-                    <mt-icon name="regular-filter" />
+                    <mt-icon
+                        name="regular-filter"
+                        size="16"
```

Comment:
`size="16"` or `:size="16"` I am wondering, because we can't bind/treat a custom prop which is in a Number type like a string.

Reply:
Alright 👍🏻